target/linux/realtek/files-5.10/Documentation/devicetree/bindings/clock/realtek,rtl8380-clock.yaml

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: GPL-2.0
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/clock/realtek,rtl8380-clock.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Realtek clock control unit Device Tree Bindings
+
+maintainers:
+  - Markus Stockhausen <markus.stockhausen@gmx.de>
+
+description: |
+  The driver provides basic clock support for the central core clock unit (CCU)
+  and its PLLs inside the RTL838X, RTL8389X and RTL930X SOC. All provided clocks
+  are derived from an oscillator that runs at a fixed speed of 25 MHz. Currently
+  switch, CPU, memory and LXB (Lexra Bus) clock information can be accessed. The
+  clocks can be referenced by 'include/dt-bindings/clock/rtl83xx-clk.h'.
+
+  If changing of clock speeds is required then SRAM must be defined via DT.
+  The relevant code is copied to SRAM and run from there. To provide SRAM
+  see http://devicetree.org/schemas/sram/sram.yaml#. E.g.
+ 
+  sram0: sram@9f000000 {
+      compatible = "mmio-sram";
+      reg = <0x9f000000 0x18000>;
+      #address-cells = <1>;
+      #size-cells = <1>;
+      ranges = <0 0x9f000000 0x18000>;
+  };
+
+  If SRAM is enabled and clock changing is allowed the driver can be directly
+  used by the DT based cpufreq driver (enabled by CONFIG_CPUFREQ_DT). At least
+  if CPU references the right clock and sane operating points (OPP) are
+  provided. E.g.
+ 
+  cpu@0 {
+      compatible = "mips,mips4KEc";
+      reg = <0>;
+      clocks = <&ccu CLK_CPU>;
+      operating-points-v2 = <&cpu_opp_table>;
+  };
+ 
+  cpu_opp_table: opp-table-0 {
+      compatible = "operating-points-v2";
+      opp-shared;
+      opp00 {
+          opp-hz = /bits/ 64 <425000000>;
+      };
+      ...
+  }
```

Comment:
You should make this a second example:
- One simple example, with a minimal DT setup
- One extended example, with all possible options added.

That way, the DT tooling will be able to detect errors in your examples too.

Please also try to verify that your binding is valid: https://www.kernel.org/doc/html/latest/devicetree/bindings/writing-schema.html (I can't spot any obvious errors)